buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -84,7 +84,7 @@ class DistributedPerformanceTest extends ReportGenerationPerformanceTest {
 
     private RESTClient client
 
-    private Map<String, Scenario> scheduledBuilds = [:]
+    protected Map<String, Scenario> scheduledBuilds = [:]
```

Comment:
This change was made because of https://builds.gradle.org/viewLog.html?buildId=21112641&buildTypeId=Gradle_Check_PerformanceFlakinessDetectionCoordinator

It seems like `DistributedPerformanceTest_Decorated` can't find private fields for dynamic invocation.